RecoParticleFlow/PFProducer/python/particleFlowEGamma_cfi.py

```diff
@@ -32,13 +32,13 @@
     # Use electron identification in PFAlgo
     usePFElectrons = cms.bool(True),
     pf_electron_output_col=cms.string('electrons'),
-    pf_electronID_mvaWeightFile = cms.FileInPath('RecoParticleFlow/PFProducer/data/MVAnalysis_BDT.weights_PfElectrons23Jan_IntToFloat.txt'),
+    pf_electronID_mvaWeightFile = cms.FileInPath('RecoParticleFlow/PFProducer/data/PfElectrons23Jan_BDT.weights.xml.gz'),
 
     # Use Photon identification in PFAlgo (for now this has NO impact, algo is swicthed off hard-coded
     usePFPhotons = cms.bool(True),
     usePhotonReg=cms.bool(False),
     useRegressionFromDB=cms.bool(True),                                 
-    pf_convID_mvaWeightFile = cms.FileInPath('RecoParticleFlow/PFProducer/data/MVAnalysis_BDT.weights_pfConversionAug0411.txt'),        
+    pf_convID_mvaWeightFile = cms.FileInPath('RecoParticleFlow/PFProducer/data/pfConversionAug0411_BDT.weights.xml.gz'),
```

Comment:
was there a good reason to rename the rest of the file (other than changing .txt to .xml.gz)?

Reply:
I will set up an externals PR with the gzipped xml weight files. The reason for the renaming was that the default naming scheme for new TMVA weights files is `*_BDT.weights.xml`. Additionally, the MVAanalysis part didn't really say much, but I can keep it in the externals PR if you like!

Then there  was this `IntToFloat` suffix which I left out, as I think replacing `I`s with`F`s in the variables section doesn't justify having basically the same weights file twice. Floats only is standard in newer TMVA anyway.

Reply:
OK. I'm not attached to the old names.